modules/gdscript/editor_templates/SCsub

```diff
@@ -2,7 +2,11 @@
 
 Import("env")
 
-import editor.template_builders as build_template_gd
+import importlib.util
+
+spec = importlib.util.spec_from_file_location("editor.template_builders", "../../../editor/template_builders.py")
```

Comment:
Might be good to add a short comment to explain this is to avoid conflict with a potential `editor` module.

Reply:
Done